components/icu/Cargo.toml

```diff
@@ -55,6 +56,7 @@ std = [
     "icu_collections/std",
     "icu_datetime/std",
     "icu_decimal/std",
+    # "icu_displaynames/std",
```

Comment:
We can use `icu_displaynames?/std` to not accidentally activate the dependency. Please update the other commented-out deps to that as well.

Reply:
Done.